explore/src/main/scala/explore/observationtree/AsterismGroupObsList.scala

```diff
@@ -339,35 +342,46 @@ object AsterismGroupObsList:
                 )
                 .orEmpty
             )(
-              ^.cursor.pointer,
-              ^.href := ctx.pageUrl(AppTab.Targets, props.programId, clickFocus),
-              ^.onClick ==> { (e: ReactEvent) =>
-                e.preventDefaultCB *> e.stopPropagationCB *> setFocused(clickFocus)
-              }
-            )(
-              csHeader,
-              TagMod.when(props.expandedIds.get.contains(obsIds))(
-                TagMod(
-                  cgObs.zipWithIndex.toTagMod { case (obs, idx) =>
-                    props.renderObsBadgeItem(
-                      ObsBadge.Layout.TargetsTab,
-                      selectable = true,
-                      highlightSelected = true,
-                      forceHighlight = isObsSelected(obs.id),
-                      linkToObsTab = false,
-                      onSelect = obsId =>
-                        setFocused(
-                          props.focused
-                            .withSingleObs(obsId)
-                            .validateOrSetTarget(obs.scienceTargetIds)
```

Comment:
What did this do? It isn't needed anymore?

Reply:
Good question, let me review it